rcl/test/rcl/test_events.cpp

```diff
@@ -730,6 +730,55 @@ TEST_F(TestEventFixture, test_bad_get_handle)
   EXPECT_EQ(NULL, rcl_event_get_rmw_handle(NULL));
 }
 
+/*
+ * Basic test subscriber event message lost
+ */
+TEST_F(TestEventFixture, test_sub_message_lost_event)
+{
+  const rmw_qos_profile_t subscription_qos_profile = default_qos_profile;
+
+  rcl_ret_t ret = setup_subscriber(subscription_qos_profile);
+  ASSERT_EQ(ret, RCL_RET_OK) << rcl_get_error_string().str;
+
+  if (is_fastrtps) {
+    // Check not supported
+    subscription_event = rcl_get_zero_initialized_event();
+    ret = rcl_subscription_event_init(
+      &subscription_event,
+      &subscription,
+      RCL_SUBSCRIPTION_MESSAGE_LOST);
```

Comment:
this code is repeated in the `if` and `else` clauses, so it would be better to put it before the `if`.

Reply:
Added with ff94914